src/components/MainCard/CardList.tsx

```
@@ -1,39 +1,58 @@
-	// console.log('meetings : ', meetings);
+	// 🟢 Zustand에서 전역 필터 상태 가져오기
+	const { selectedFilters } = useFilterStore();
 
+	// ✅ 모임 데이터 가져오기
 	const getMeetingListDate = async () => {
 		try {
 			const res = await getMeetingData();
 			setMeetings(res);
 		} catch (error) {
-			console.error('모임 목록 가져오기 기능 실패 : ', error);
+			console.error('모임 목록 가져오기 실패:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)